### DISCUSSION TITLE
请问有什么好的办法，在按下R键开始录音练习时能够准确的知道软件已经开始在同步录音？

### DISCUSSION THREAD

**Discussion by chinecc** [ZuodaoTech/everyone-can-use-english#1058] [Q&A]
目前遇到一个比较头疼的问题，
经常按下R键录音后，
句子最开头的几秒钟是没有被记录下来的，
导致和原声进行比较时，
不能很方便的进行比较？
不知道有什么办法可以解决这个问题？
比如按下R键后，有一个提示音，表示录音正式开始之类的？

**Comment by an-lee** [comment]
几秒？你在用 Enjoy 的版本号多少？

**Comment by an-lee** [comment]
能录个视频看看吗？这个不正常

**Reply by chinecc** [reply]
我已经升级到了0.5.0版本也遇到一样的情况。主要的问题就是按下R键后，我直接说话录音，结束录音后，播放出来前面几秒是没有录进来的。除非我一开始先停顿一会儿在开始说话，可以录完整，但是和原生的语音就不能够很方便的进行一对一比较。
我用的是mac版本，蓝牙AirPods pro录音。
另外我在软件的设置里面已经关掉了降燥之类的开关，
也仍然有问题。
我抽个时间试下能不能录个视频把问题重现出来给你，
非常感谢你

**Comment by chinecc** [comment]
https://github.com/user-attachments/assets/98195de3-5bec-4c68-b2e4-8be9f4805362

@an-lee 大佬，我大概录了一下我的这个问题的重现过程，
你帮忙看一下，是否有解决办法？

严格来说应该不是软件录音的问题，
是我按下R键后，很难把握住开始录音的点，
最后就会导致前面有时候没录进来有时候是往后才开始录，
如果有一个录音倒数计时之类的，
我想应该就方便很多。

**Reply by an-lee** [reply]
没看明白这个视频，录完之后得到的录音是咋样的？不正常吗？

**Comment by chinecc** [comment]
![QQ_1726498748922](https://github.com/user-attachments/assets/c93355a8-fe1a-4c22-9031-c12968478bc4)
@an-lee 你好，就是录音开头这里，很难把握住开始录音的点，看截图这里最前面那一段是还没说话时就被录下来了，所以导致和下面的原声不能直接快速的进行对比。
录音功能是没有问题的，
我是想问有没有解决办法能够有效的从开始发出第一个单词的声音开始记录？
我有时候是遇到录下来的声音前面几个单词没有存下来，
有时候是现在这个视频里的情况，前面没说话时的声音被记录下来，不好直接对比。

**Reply by an-lee** [reply]
大概理解你的意思了。可以试试用耳机，这样可以减少噪音。然后录音前面空一段没关系的，用`对比`功能的时候会自动截掉前后空白的录音，如果环境噪声比较大，前后空白录音的自动截断可能会失效。
